checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -462,6 +465,7 @@ public int hashCode() {
     this.typeFactory = typeFactory;
     this.checker = (ResourceLeakChecker) typeFactory.getChecker();
     this.analysis = analysis;
+    this.permitInitializationLeak = checker.hasOption("permitInitializationLeak");
```

Comment:
What is the point of using a static final constant?  From my point of view, it:
 * adds indirection
 * adds clutter
 * makes it harder to search for all uses of the flag in the code (because the static constant name is spelled differently from the command-line switch itself)
 * gives no help if we ever want to change the name.
    * it is trivial to do a global search
    * we must do a global search anyway, to find occurrences in comments and in the manual

I don't see any benefits to using a static final constant.

Reply:
> makes it harder to search for all uses of the flag in the code

If you're using an IDE, it should actually make it easier - a global search for uses of the static final constant will return only the places where it is used in the code.

Regardless, this is a matter of style/preference. Since you wrote this code, I'm fine with it.